classes/Dispatcher.php

```diff
@@ -805,7 +805,7 @@ public function hasRoute($route_id, $id_lang = null, $id_shop = null)
             $id_shop = (int) Context::getContext()->shop->id;
         }
 
-        return isset($this->routes[$id_shop], $this->routes[$id_shop][$id_lang], $this->routes[$id_shop][$id_lang][$route_id])
+        isset($this->routes[$id_shop][$id_lang][$route_id])
```

Comment:
No more return? :D and `;` at the same line?

Reply:
Woops :/